docs/docs/references/tables.mdx

```diff
@@ -161,6 +161,10 @@ Use `required_filters` to define filters on Dimensions that will be applied when
 This allows to enforce a default filter on a Dimension when querying a table.
 This defined `required_filters` will be replaced by any user-defined filter on the same Dimension making sure at least one filter is applied to the Dimension.
 
+This is particularly useful when you want to enforce a default filter to save on query time and potential warehouse costs.
+
+> **Note**: A required filter's field reference can't be changed, but its operator(is/is not,etc) and value can be changed when querying the table.
```

Comment:
@joaoviana we should use an info block here instead, [like this one](https://github.com/lightdash/lightdash/blob/3c005f085ce1461492b71145a384c1361c272f06/docs/docs/references/metrics.mdx?plain=1#L550-L568).

Also small grammar change: but its operator (is, is not, etc.)

Reply:
ty! 
<img width="1041" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/9e113b21-f373-4c96-937d-de3d7f496288">

done @jpetey75